apache-apisix/manifest.json

```diff
@@ -26,7 +26,7 @@
     "support_email": "dev@apisix.apache.org",
     "homepage": "https://github.com/DataDog/integrations-extras",
     "sales_email": "dev@apisix.apache.org",
-    "name": "unknown"
+    "name": "Community"
```

Comment:
Just curious, but is `Community` just applied to all non-partners? I checked the [codeowner](https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L15) for `apache-apisix` and looks like the [contributor](https://github.com/bisakhmondal) does work for Apache APISIX.

Reply:
Resolved via slack!